pkg/controller/hibernation/hibernation_controller_test.go

```diff
@@ -172,7 +172,7 @@ func TestReconcile(t *testing.T) {
 			cd:   cdBuilder.Options(o.shouldHibernate, o.stopping).Build(),
 			cs:   csBuilder.Build(),
 			setupActuator: func(actuator *mock.MockHibernationActuator) {
-				actuator.EXPECT().MachinesStopped(gomock.Any(), gomock.Any(), gomock.Any()).Times(1).Return(true, nil)
+				actuator.EXPECT().MachinesStopped(gomock.Any(), gomock.Any(), gomock.Any()).Times(1).Return(true, nil, nil)
```

Comment:
nit: As noted above, `MachinesStopped` will never return `nil` for the second return value (except in error cases). It totally doesn't affect anything, but if you wanted to be pedantic:
```suggestion
				actuator.EXPECT().MachinesStopped(gomock.Any(), gomock.Any(), gomock.Any()).Times(1).Return(true, []string{}, nil)
```
Same applies for `MachinesRunning` below.

Reply:
for consumers of slices, both should have the same behavior. So i would rather just keep it nil for clarity sake.